spec/services/staff_nomis_checker_spec.rb

```diff
@@ -174,6 +174,10 @@
       let(:slot) { visit.slots.first }
 
       context 'prisoner availability' do
+        before do
```

Comment:
Same thing some ENV aren't stubbed and and based on your .env tests start failing